Framework/PythonInterface/plugins/algorithms/CylinderPaalmanPingsCorrection2.py

```diff
@@ -405,10 +405,10 @@ def _set_material(self, ws_name, chemical_formula, density_type, density):
             mat = builder.setFormula(chemical_formula).setMassDensity(density).build()
```

Comment:
I agree `builder` is no longer required, but I think the extra lines are still needed as the algo returns the number density.